EPlast/EPlast.BLL/Services/Auth/AuthService.cs

```diff
@@ -241,6 +246,59 @@ public async Task SendEmailReseting(string confirmationLink, ForgotPasswordDto f
                 $"Для скидування пароля перейдіть за : <a href='{confirmationLink}'>посиланням</a>", "Адміністрація сайту EPlast");
         }
 
+        ///<inheritdoc/>
+        public async Task<User> GetGoogleUserAsync(string providerToken)
+        {
+            string GoogleApiTokenInfoUrl = "https://www.googleapis.com/oauth2/v3/tokeninfo?id_token={0}";
+            var httpClient = new HttpClient();
+            var requestUri = new Uri(string.Format(GoogleApiTokenInfoUrl, providerToken));
+
+            HttpResponseMessage httpResponseMessage = httpClient.GetAsync(requestUri).Result;
```

Comment:
```suggestion
            HttpResponseMessage httpResponseMessage = await httpClient.GetAsync(requestUri)
```
Why don't we use this?